samples/nrf9160/secure_services/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _secure_services:
+
+nRF9160: Secure Services Sample
+###############################
+
+The Secure Services sample shows how to use the secure services provided by :ref:`secure_partition_manager`.
+This firmware needs :ref:`secure_partition_manager` to also be present on the chip.
+
+Overview
+********
+
+This sample will generate random numbers and print them to the console, sleep, then reboot.
```

Comment:
generates, instead of will generate to have the sentence in simple present tense. I don't know if this convention.

Reply:
Yep, I like that better. Done.